DicesGame/Dices.cs

```diff
@@ -16,8 +16,11 @@ public partial class Dices : Form
         private List<Image> _diceFaceImages;
         private IDiceFactory _factory;
         private GameRule _gameRule;
-        private int _roundNumber = 0;
+        private int _roundNumber = 1;
         private int counter = 0;
+        private int gameWinner;
+        private Random random1;
+        private Random random2;
 
         public Dices()
```

Comment:
maybe you can create a function like InitVariables and just call it in the constructor ?